### DISCUSSION TITLE
Particles in Athena++ ?

### DISCUSSION THREAD

**Discussion by wadsley** [PrincetonUniversity/athena#572] [General]
I have been using old Athena (C) to simulate dust in shearing boxes.   I would like to take advantage of the adaptivity in Athena++ and do similar things.   I would be willing to code a bit if needed but I'd prefer to start from an existing particle type in the code.    To be specific, I'd like to do dust in shearing boxes with self-gravity.    The documentation does not mention particles but I thought there might be some experimental work or a bit of a basic framework for particles already in existence somewhere.

**Comment by chaochinyang** [comment]
Dear Dr. Wadsley,

There are two versions of dust particles in development.  I am developing the Lagrangian type, while Pinghui Huang & Xuening Bai ([2022](https://doi.org/10.3847/1538-4365/ac76cb)) are developing the Eulerian fluid type. However, the Lagrangian dust particles are not ready for shearing box yet; on the other hand, the fluid dust should be available for shearing box.
More seriously, self-gravity in shearing box also needs quite some attention in development.  Kazunari Iwasaki and @tomidakn may have more information on this front.

Sincerely,
Chao-Chin
